### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_AzureNotebooksWelcome_Keyboard: Keyboard focus is not Shifting to link (i.e table of contents) controls present on page.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3934]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to run button & hit enter --> Navigate to Azure Notebooks    
    welcome & hit enter.
3. Start Navigating through Menubar using keyboard.
4. Verify Keyboard focus is not Shifting to to link (controls) present on page.

**Actual:**
Unable to navigate using tab key on page.
Keyboard focus is not Shifting to link (controls) present on page.

Same issue observed on "Chrome + NVDA " configuration.

**Expected:**
User should navigate on all controls using tab key.

**UserImpact:**
Keyboard Only Users face difficulty if Keyboard focus is not Shifting to actionable controls.

**Comment by AccessibilityTestingTeam-TCS** [comment]
For more details please find attached video.

**Comment by AccessibilityTestingTeam-TCS** [comment]
For more details please find attached video/powerpoint slide.
[3934_A11y_AzureNotebooks_Jupyter_AzureNotebooksWelcome_Keyboard.pptx](https://github.com/jupyter/notebook/files/2396655/3934_A11y_AzureNotebooks_Jupyter_AzureNotebooksWelcome_Keyboard.pptx)

**Comment by minrk** [comment]
All of these issue reports are great! Should we be expecting pull requests to help fix some of them?

**Comment by jasongrout** [comment]
There is some discussion offline about others starting to tackle some of this work.

**Comment by jasongrout** [comment]
(but if someone wants to start working on these, by all means go for it!)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
